.github/workflows/pull-request.yaml

```diff
@@ -21,6 +21,7 @@ jobs:
       tox-version: ""
       python-version: "3.10"
 
-  check-file-updates:
+  update-file:
     name: Check updates for charm files
-    uses: ./.github/workflows/_check_file_update.yaml
+    uses: ./.github/workflows/update_file.yaml
```

Comment:
suggestion: we are updating multiple files so this could be named more appropriately. For example "sync-from-sunbeam"

Reply:
unresolving as this also applies to line 24

Reply:
and line 2 of the workflow file